scooter/src/fields.rs

```diff
@@ -280,12 +280,20 @@ impl Field {
         }
     }
 
-    fn create_title_spans<'a>(&self, title: &'a str, highlighted: bool) -> Vec<Span<'a>> {
-        let title_style = Style::new().fg(if highlighted {
-            Color::Green
-        } else {
-            Color::Reset
-        });
+    fn create_title_spans<'a>(
+        &self,
+        title: &'a str,
+        highlighted: bool,
+        set_by_cli: bool,
+        disable_populated_fields: bool,
+    ) -> Vec<Span<'a>> {
+        let mut fg_color = Color::Reset;
+        if set_by_cli && !disable_populated_fields {
```

Comment:
I think this should be

```suggestion
        if set_by_cli && disable_populated_fields {
```